CHANGELOG.md

```diff
@@ -117,6 +114,19 @@ Adding a new version? You'll need three changes:
   time but be aware that your mileage may vary).
   [#4222](https://github.com/Kong/kubernetes-ingress-controller/pull/4222)
 
+[gojson]: https://github.com/goccy/go-json
+
+## [2.10.2]
+
+> Release date: 2023-07-06
```

Comment:
```suggestion
> Release date: 2023-07-07
```



I'm not sure if we shouldn't perhaps fix that one :) (similarly on the release branch ..sigh)